src/components/Signup/Signup.test.js

```diff
@@ -6,13 +6,24 @@ import { MemoryRouter as Router } from 'react-router-dom';
 import { AppServicesProvider } from '../../services/pure-di';
 import 'jest-dom/extend-expect';
 
+class MenubuttonFake {
+  // eslint-disable-next-line no-useless-constructor
+  constructor() {}
```

Comment:
Maybe declaring the constructor is not necessary, could you try removing it?

Reply:
Sure, I thought I needed this since the tests gave an error about a constructor missing. I've tested and it is not neccesary.